assets/messages.yml

```diff
@@ -108,6 +108,39 @@ messages:
         *Thank you for your report!*
         Could you please include information about your server? Without it we may be unable to reproduce the issue.
 
+        %quick_links%
+      fillname: []
+  attach-video:
+    - project:
+        - mc
+        - mcl
+        - mcpe
+        - mcd
+      name: Attach video (PC)
+      message: |-
+        _We do not have enough information to find the cause of this issue._
+        
+        Please *record a video of this happening* and attach it to this report.
+        If you are on Windows, you can use {{Windows}}+{{Alt}}+{{R}} to open a built-in App for recording game footage.
+        In case you don''t have a program to record videos, we recommend using the free recording software [OBS|https://obsproject.com/].
```

Comment:
```suggestion
        In case you don't have a program to record videos, we recommend using the free recording software [OBS|https://obsproject.com/].
```

Reply:
This is an escaped apostrophe in yaml and I changed this intentionally.

Reply:
I have merged your commit into my fork (deployed at https://marcono1234.github.io/helper-messages/#MC) and there the duplicate `'` appears. Literal scalars in YAML apparently do not support escaping: https://yaml.org/spec/1.2/spec.html#style/block/literal
Syntax highlighting is probably wrong then.

Reply:
Does my fork break something else?
There is a weird orange background for me: 
![image](https://user-images.githubusercontent.com/12124394/82229716-1862e800-992b-11ea-92a8-09db442a5b6c.png)